composer.json

```
@@ -33,6 +33,7 @@
         "phpstan/phpstan": "1.12.7",
         "phpunit/phpunit": "10.5.36",
         "phing/phing": "3.0",
+        "rector/rector": "^1.2",
```

Do you mind pinning this to the specific version before we merge?

```suggestion
        "rector/rector": "1.2.7",
```